app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%= render partial: "layouts/head" %>
+  </head>
+  <body class="app-theme">
+    <a class="skip sr sr-focusable" href="#main-menu">Skip to main menu</a>
```

Comment:
Our skip links need `id=skip` to work... which we can't do because that would lead to multiple conflicting `ids`.

This discussion started on Slack, but not to the point of realizing (or at least I didn't realize at the time) that supporting multiple skip links will introduce a breaking change if we move from `id=skip` to `class=skip` at the theme gem level. We likely want to deprecate `id=skip` and move to to a new class name, such as `class=skiplink` to allow us to slowly transition all of our apps to the new behavior. All of the rails apps will get the new markup at the same time as they get the new CSS, but I believe omeka directly pulls the style guide CSS so we should take care to not break things when adjusting this feature at our style guide level.

We also noted that our current skip link CSS is broken in Safari. We should resolve that when moving from `id` to `class` to support this new feature. If you'd prefer to add timdex-ui css overrides for these changes and open a ticket to move them to the style guide/theme gem once we work out any kinks in this app that would be fine with me.